Add information in sliding-window-counts-plugin.md (#1714)
MicrosoftDocs/dataexplorer-docs: data-explorer/kusto/query/sliding-window-counts-plugin.md

| *TimelineColumn* | string | &check; | The name of the column representing the timeline.|
| *Start* | int, long, real, datetime, or timespan | &check; | The analysis start period.|
| *End* | int, long, real, datetime, or timespan | &check; | The analysis end period.|

> | *LookbackWindow* | int, long, real, datetime, or timespan | &check; | The lookback period, this value should be a multiple of the *Bin* value, if not the *LookbackWindow* value will be replaced by a multiple of the *Bin* value. For example, for `dcount` users in past `7d`: *LookbackWindow* = `7d`.|

Reviewer: ```suggestion
| *LookbackWindow* | int, long, real, datetime, or timespan | &check; | The lookback period. This value should be a multiple of the *Bin* value, otherwise the value will be rounded down to a multiple of the *Bin* value. For example, for `dcount` users in past `7d`: *LookbackWindow* = `7d`.|
```
